service/src/main/kotlin/fi/espoo/evaka/assistanceneed/vouchercoefficient/AssistanceNeedVoucherCoefficient.kt

```diff
@@ -15,6 +16,8 @@ data class AssistanceNeedVoucherCoefficient(
     val childId: ChildId,
     val coefficient: BigDecimal,
     val validityPeriod: FiniteDateRange,
+    val modifiedAt: HelsinkiDateTime?,
+    val modifiedBy: String?,
```

Comment:
I suppose it could hypothetically be re-written that way, but for this change I'm not sure there's much benefit to returning more than the name from the DB query. Is there an existing convention to return the entire evaka_user row, though?

Reply:
Or you could rename this into `modifiedByName` and keep this as String. And then change all functions in `AssistanceNeedVoucherCoefficientQueries` to join the user name from the evaka_user.